### DISCUSSION TITLE
Question about adding new boards during HWMv2 efforts

### DISCUSSION THREAD

**Discussion by piotrrak** [zephyrproject-rtos/zephyr#69188] [General]
## Introduction

Hello

Late eveneing I've managed to port blinky to Renesas EK-RA6M5 (Cortex-M33) board.
It still needs work on clocks and stuff since clock generation is bit more complicated on RA6, RA8 series of MPUs,
and bit of cleanup in dts files/ documentation.
I am also planing to later work on contribute later:
- ~EK-RA6M3 (Cortex-M4)~ EK-RA4M3 (Cortex-M33)
- EK-RA8M1 (Cortex-M85) - this was what I've started with but switched to M33 to not learn before I add new Core and toolchain
- EK-RA8T1 (Cortex-M85) - mostly interested in that kit with 3-phase inverter bridge

Possibly also EK-RA8D1 (almost like RA8M1 really not much difference beside pheripherials) 
I should have access to one in near future.

I've noticed https://github.com/zephyrproject-rtos/zephyr/pull/69176 and similar PRs (also related to code I am basing on my work - already supported Arduino Uno R4 is Renesas  RA series)

Is there any place with informations about this project/changes I could read about?
Are there any things I should be aware of that are comming up?
Or, should I simply stay with master and ignore that and port later?

Cheers,
P

**Comment by dbaluta** [comment]
Maybe you can start with this? https://github.com/zephyrproject-rtos/zephyr/issues/51831

**Reply by piotrrak** [reply]
@dbaluta  Thank you

**Comment by nordicjm** [comment]
Documentation on hwmv2: https://github.com/zephyrproject-rtos/zephyr/pull/68792

hwmv1 boards will no longer be accepted upstream, submissions must be in hwmv2

**Reply by piotrrak** [reply]
@nordicjm Thank you. Will take a look. Got that, I've submitted my WIP rewritten on top of your work  with RA `hwmv2-arm-renesas` here: https://github.com/zephyrproject-rtos/zephyr/pull/69315 I've noticed it was already merged. If you wish me to seperate some of those changes to smaller commits or submit bug fixes to main please let me know.

The more I look at it the more work I see, so it looks it might grow to quite substantial change comming weeks/months.

As I've mentioned I am mostly interested in getting  [RA8T1-MTK](https://www.renesas.com/us/en/products/microcontrollers-microprocessors/ra-cortex-m-mcus/rtk0ema5k0s00020bj-mck-ra8t1-renesas-flexible-motor-control-kit-ra8t1-mcu-group) dev board up and running, though I got myself some of cheaper EK-RA series boards to aid me in that goal and possibly be usable in future projects.

I don't plan to use Renesas FSP so will be writing some of the drivers that will be usable in zephyr anyway.

Last of my changes DR is worth looking at, since it's probably a bug fix, though I don't have arduino and can't confirm it.
